Student/StudentUITests/Assignments/AssignmentDetails/AssignmentDetailsTests.swift

```diff
@@ -275,6 +283,14 @@ class AssignmentDetailsTests: StudentUITestCase {
         XCTAssertEqual(AssignmentDetails.gradeLatePenalty.label, "Late penalty (-5 pts)")
     }
 
+    func testViewSubmissionButtonHiddenWhenNotGraded() {
+        let assignment = mockAssignment(APIAssignment.make(
+            submission_types    : [.not_graded]
+        ))
+        show("/courses/\(course.id)/assignments/\(assignment.id)")
+        XCTAssertFalse(AssignmentDetails.viewSubmissionButton.isVisible)
```

Comment:
I think this conflicts with some of the other tickets. I think we need this to always be visible, since it's how you get to comments and rubric. Even when there is no submission type, there can be comments.

Reply:
Ah I see. I was trying to match web in preventing students from viewing submissions but rubrics are indeed exposed to students on the assignment details on web. I will revert this part.